lambdas/evaluate-gpg45-scores/src/main/java/uk/gov/di/ipv/core/evaluategpg45scores/EvaluateGpg45ScoresHandler.java

```diff
@@ -134,23 +124,17 @@ public Map<String, Object> handleRequest(JourneyRequest event, Context context)
                     gpg45ProfileEvaluator.parseCredentials(
                             userIdentityService.getUserIssuedCredentials(userId));
 
-            if (!checkCorrelation(userId)) {
+            if (!userIdentityService.areVcsCorrelated(userId)) {
                 return JOURNEY_PYI_NO_MATCH.toObjectMap();
             }
 
-            Optional<JourneyResponse> journeyResponseForFailWithNoCi =
-                    getJourneyResponseForFailWithNoCi(ipvSessionItem, userId);
-            if (journeyResponseForFailWithNoCi.isPresent()) {
-                return journeyResponseForFailWithNoCi.get().toObjectMap();
-            }
-
             return checkForMatchingGpg45Profile(
                             userId, ipvSessionItem, clientOAuthSessionItem, credentials, ipAddress)
                     .toObjectMap();
         } catch (HttpResponseExceptionWithErrorBody e) {
             LOGGER.error("Received HTTP response exception", e);
             return new JourneyErrorResponse(
-                            JOURNEY_ERROR_PATH, e.getResponseCode(), e.getErrorResponse())
+                            JOURNEY_ERROR_PATH, HttpStatus.SC_BAD_REQUEST, e.getErrorResponse())
```

Comment:
Why has this changed?

Reply:
It works in the context, since any errors come from the request. That said, I meant to revert this change, thanks for the spot